packages/design-system/src/components/ChoiceList/ChoiceList.test.tsx

```diff
@@ -26,143 +26,169 @@ function render(customProps = {}, choicesCount = 2, deep = true) {
     },
     ...customProps,
   };
-  const component = <ChoiceList {...props} />;
-
-  return {
-    props,
-    wrapper: deep ? mount(component) : shallow(component),
-  };
+  return render(<ChoiceList {...props} />);
 }
 
 describe('ChoiceList', () => {
   describe('Radio buttons and Checkboxes', () => {
+    afterEach(() => {
+      jest.clearAllTimers();
+      jest.useRealTimers();
+    });
+
     it('is a radio button group', () => {
-      const { wrapper } = render({ type: 'radio' });
+      const { queryByLabelText, asFragment } = renderChoiceList();
+      const choiceEl = queryByLabelText('Choice 1') as HTMLInputElement;
 
-      expect(wrapper.find('Choice').first().prop('type')).toBe('radio');
-      expect(wrapper).toMatchSnapshot();
+      expect(choiceEl.type).toBe('radio');
+      expect(asFragment()).toMatchSnapshot();
     });
 
     it('is a checkbox group', () => {
-      const { wrapper } = render({ type: 'checkbox' });
+      const { queryByLabelText } = renderChoiceList({ type: 'checkbox' });
+      const choiceEl = queryByLabelText('Choice 1') as HTMLInputElement;
 
-      expect(wrapper.find('Choice').first().prop('type')).toBe('checkbox');
+      expect(choiceEl.type).toBe('checkbox');
     });
 
     it('is a checkbox', () => {
-      const { wrapper } = render({ choices: generateChoices(1), type: 'checkbox' });
+      const { queryByLabelText } = renderChoiceList({
+        choices: generateChoices(1),
+        type: 'checkbox',
+      });
+      const choiceEl = queryByLabelText('Choice 1') as HTMLInputElement;
 
-      expect(wrapper.find('Choice').first().prop('type')).toBe('checkbox');
+      expect(choiceEl.type).toBe('checkbox');
     });
 
     it('renders all choices', () => {
-      const { props, wrapper } = render();
-      const choice = wrapper.find('Choice').first();
+      const numChoices = 3;
+      const { queryAllByRole } = renderChoiceList({}, numChoices);
```

Comment:
The other day when I was troubleshooting the tests I was creating, I was looking through this article and saw this heading: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen. KCD recommends using screen over the destructured `render` return value. It's a tiny thing, but I thought I'd share that.

Reply:
Thanks for sharing! This had some good suggestions for several updates that I've now incorporated to the tests